scaleway/helpers_rdb_test.go

```diff
@@ -0,0 +1,115 @@
+package scaleway
+
+import (
+	"net"
+	"testing"
+
+	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsEndPointEqual(t *testing.T) {
+	tests := []struct {
+		name     string
+		A        *rdb.EndpointPrivateNetworkDetails
+		B        *rdb.EndpointPrivateNetworkDetails
+		expected bool
+	}{
+		{
+			name: "isEqualPrivateNetworkDetails",
+			A: &rdb.EndpointPrivateNetworkDetails{PrivateNetworkID: "6ba7b810-9dad-11d1-80b4-00c04fd430c8", ServiceIP: scw.IPNet{IPNet: net.IPNet{
+				IP: net.IPv4(0x1, 0x1, 0x1, 0x1), Mask: net.IPMask{0xff, 0xff, 0xff, 0xff}}}, Zone: scw.ZoneFrPar1},
```

Comment:
Use integer notation and have a field per line, it would be easier to read and would allow to have an helpful diff